intelmq/lib/mixins/http.py

```diff
@@ -65,8 +65,8 @@ def setup(self):
         # tls certificate settings
         if self.ssl_client_cert is not None:
             self.__session.cert = self.ssl_client_cert
-        # auth settings
-        if self.http_username is not None:
+        # auth settings. username or password must exist (if only one, then this is likely an error, but we should try without auth) and not be an empty string
+        if self.http_username or self.http_password:
```

Comment:
would it work only with a pwd? (no username?)

Reply:
Not sure if I understand. What change do you request here?

Reply:
Well a pure password in http auth without username is meaningless.. no? How do HTPT servers react?